open-ce/utils.py

```diff
@@ -219,6 +219,74 @@ def cuda_driver_installed():
 
         raise OpenCEError(Error.ERROR, "lsmod command unexpectedly failed") from err
 
+def check_cuda_version_match():
+    '''
```

Comment:
This function should be checking that the value of `command.cudatoolkit` is equivalent to the version of CUDA that is installed within the CUDA_HOME environment variable.

Reply:
Changed to check version from `command.cudatoolkit` with version within CUDA_HOME (checked via what is reported in the `$CUDA_HOME/version.txt` file).  Now, issue a WARNING if we can't get the version.